collector/lib/CollectorConfigInspector.cpp

```diff
@@ -0,0 +1,61 @@
+#include "CollectorConfigInspector.h"
+
+#include <forward_list>
+#include <string>
+#include <unordered_map>
+
+#include <arpa/inet.h>
+
+#include <google/protobuf/util/json_util.h>
+
+#include "ConnTracker.h"
+
+namespace collector {
+
+const std::string CollectorConfigInspector::kBaseRoute = "/state/config";
+
+CollectorConfigInspector::CollectorConfigInspector(const std::shared_ptr<CollectorConfig> config) : config_(config) {
+}
+
+Json::Value CollectorConfigInspector::configToJson() {
```

Comment:
There are a few situations in this method that could be considered errors, like failing to convert the configuration to JSON, it'd be nice if we could change the HTTP status to 500 if one of these occur, maybe we can return a tuple with the result and the json? Or use a `std::variant` of sorts?

Reply:
Done